src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -65,24 +64,20 @@ public void deleteLineById(Long id) {
     }
 
     private Line findLineById(Long id) {
-        return lineRepository.findById(id).orElseThrow(BadRequestException::new);
+        return lineRepository.findById(id).orElseThrow(() -> {
```

Comment:
```suggestion
        return lineRepository.findById(id)
                .orElseThrow(() -> new BadRequestException("존재하지않는 노선 ID 입니다."));
```

1. 한 줄에 한 점만 찍어야 코드 의미 파악에 좋습니다.
2. orElseThrow를 잘못 사용하고 있습니다. 관련하여 전체 테스트 돌리면 컴파일 에러가 발생하니 같이 점검해보아요

Reply:
넵 감사합니다~!